### PR TITLE
Add stripPunctuation method

### PR DESCRIPTION
**Pull request by wecerny**

### CODE REVIEW

**Comment by wecerny** [comment]
Looks like PHP up to version 5.6 interprets the **№** sign as non-ascii character

File: tests/StringyTest.php [hunk 1]
```diff
@@ -2482,4 +2482,24 @@ public function htmlDecodeProvider()
...
+        $this->assertStringy($result);
+        $this->assertEquals($expected, $result);
+        $this->assertEquals($str, $stringy);
+    }
+
+    public function stripPunctuationProvider()
+    {
+        return array(
+            array('foo bar ', '!{foo}:,+"`-=\/ (&bar.) @#'),
+            array('fòô bàř 123', '!{fòô}: (bàř)? №1;2--__3%', 'UTF-8'),
```
Comment (danielstjules) [inline tests/StringyTest.php]: Isn't this stripping a lot more than just punctuation? https://en.wikipedia.org/wiki/Punctuation#In_computing

Comment (wecerny) [inline tests/StringyTest.php]: General idea was to be able to wipe out all non alphanumeric chars from string, leaving words and numbers only. I can rename this method somehow, and then dig deeper in the field of punctuations to add relevant methods.

Comment (wecerny) [inline tests/StringyTest.php]: Any suggestions?

Comment (danielstjules) [inline tests/StringyTest.php]: Maybe strip symbols would be more appropriate?

**Pull request closed without merging**